include/tvm/contrib/gemm.h

```diff
@@ -0,0 +1,94 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file tvm/contrib/gemm.h
+ * \brief Shared implementation of gemm
+ */
+#ifndef TVM_CONTRIB_GEMM_H_
+#define TVM_CONTRIB_GEMM_H_
+#include <algorithm>
+
+namespace tvm {
+namespace contrib {
+
+  using namespace runtime;
+
+  inline int columnStride(DLTensor* tensor) {
```

Comment:
some note about style:

- do not put auxiliary function header in inlcude, these features are better suited in the private part, so put them in src/contrib/gemm

- Use CamelCase for functions(google C style)